pkg/apis/monitoring/v1alpha1/validation.go

```diff
@@ -61,6 +61,12 @@ func (hc *HTTPConfig) Validate() error {
 		}
 	}
 
+	if hc.ProxyConnectHeader != nil {
```

Comment:
we need the same addition to pkg/apis/monitoring/v1beta1/validation.go

Reply:
Thanks. Fixed it done.